### ISSUE TITLE
Is there any in build functionality for edit using popup like add.

### ISSUE THREAD

**Issue by ManrajParmarSipl** [AllenFang/react-bootstrap-table#1686]
If Is there any in build functionality for edit using popup like add then please share documentation for that.

**Comment by AllenFang** [comment]
@ManrajParmarSipl no, but this will be a new feature on `react-bootstrap-table2` but it still work in progress

https://github.com/react-bootstrap-table/react-bootstrap-table2
